00-open-a-shapefile.Rmd

```diff
@@ -8,9 +8,9 @@ contributors: [Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 packagesLibraries: [rgdal, raster]
-category: [self-paced-tutorial]
+category: 
```

Comment:
What lesson do you see it in? I agree but I thought I had removed it from all lessons. looking at the code in this repo I can't find it! Please let me know.

Please excuse any typos - sent from my Android.

-----Original Message-----
From: Megan A. Jones [notifications@github.com]
Received: Wednesday, 27 Jan 2016, 16:38
To: data-lessons/NEON-R-Spatial-Vector [NEON-R-Spatial-Vector@noreply.github.com]
CC: Leah Wasser [lwasser@neoninc.org]
Subject: Re: [NEON-R-Spatial-Vector] _include file update and lesson edits (#75)


In 00-open-a-shapefile.Rmd<https://github.com/data-lessons/NEON-R-Spatial-Vector/pull/75#discussion_r51068856>:

> @@ -8,9 +8,9 @@ contributors: [Sarah Newman]
>  dateCreated:  2015-10-23
>  lastModified: `r format(Sys.time(), "%Y-%m-%d")`
>  packagesLibraries: [rgdal, raster]
> -category: [self-paced-tutorial]
> +category:


I believe vector-data-workshop should also be removed as a tag from the other lessons.

-
Reply to this email directly or view it on GitHub<https://github.com/data-lessons/NEON-R-Spatial-Vector/pull/75/files#r51068856>.